notebooks/223-text-prediction/223-text-prediction.ipynb

```diff
@@ -0,0 +1,773 @@
+{
```

Comment:
<p>not clear what is it mean, could you please add description?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/897/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>